packages/docs/docs/tutorial/get-started/create-and-integrate-the-first-app.mdx

```diff
@@ -0,0 +1,35 @@
+---
+sidebar_position: 2
+---
+
+import Columns from '@components/Columns';
+
+# Create and integrate the first application
+
+Click the "Create" button on the right, and the browser will redirect to the Application tab. Then click the "Create Application" button on this page.
+
+**Step 1**
+
+In the opening modal, choose which type of application you'd like to integrate:
+
+- **Native App** is an app that runs in a native environment, e.g., iOS app, Android app.
+- **Single Page App** is an app that runs in a web browser, which updates the page with the new data from the server without loading entire new pages. E.g., React DOM app, Vue app.
+- **Traditional Web App** is an app that renders and updates pages by the web server alone. E.g., JSP, PHP.
+
+We prepared different tutorials for each application type, and it'll be helpful if you choose the type properly.
```

Comment:
nit: and it'll be helpful if you choose the type properly => `Please choose the type properly in order to better assist your integration`

Reply:
kind of tough to use "please"?

Reply:
No... I'm just trying to be `polite` 😂

Reply:
usually please is not polite :-)

Reply:
Well, maybe I'm just being old-fashioned. lol
I'm good with this